packages/components/src/components/navitem/navitem.styles.ts

```diff
@@ -0,0 +1,105 @@
+import { css } from 'lit';
+import { hostFitContentStyles, hostFocusRingStyles } from '../../utils/styles';
+
+const styles = [hostFitContentStyles, css`
+  :host {
+    --mdc-navitem-color: var(--mds-color-theme-text-primary-normal);
+    --mdc-navitem-border-color: var(--mds-color-theme-outline-button-normal);
+    --mdc-navitem-hover-background-color: var(--mds-color-theme-button-secondary-hover);
+    --mdc-navitem-pressed-background-color: var(--mds-color-theme-button-secondary-pressed);
+    --mdc-navitem-disabled-background-color: var(--mds-color-theme-button-secondary-disabled);
+    --mdc-navitem-disabled-color: var(--mds-color-theme-text-primary-disabled);
+    --mdc-navitem-active-background-color: var(--mds-color-theme-button-secondary-active-normal);
+    --mdc-navitem-expanded-width: 12.75rem;
+ 
+    position: relative;
+    display: flex;
+    align-items: center;
+    gap: 0.5rem;
+    padding: 0.5rem;
+    color: var(--mdc-navitem-color);
+    border-color: var(--mdc-navitem-border-color);
+    border-radius: 1.25rem;
+    margin-left: 1rem;
+    cursor: pointer;
+  }
+
+  :host(:dir(rtl)) {
+    margin-right: 1rem;
+  }
+
+  :host([disabled]) {
+    pointer-events: none;
+  }
+
+  :host([isExpanded]) {
+    width: var(--mdc-navitem-expanded-width);
+  }
+  
+  :host(:hover), 
+  :host(:hover[active]) {
+    background-color: var(--mdc-navitem-hover-background-color);
+  }
+
+  :host(:active) {
+    background-color: var(--mdc-navitem-pressed-background-color);
+  }
+
+  :host([disabled]), 
+  :host([active][disabled]) {
+    color: var(--mdc-navitem-disabled-color);
+    background-color: var(--mdc-navitem-disabled-background-color);
+  }
+
+  :host([active]) {
+    background-color: var(--mdc-navitem-active-background-color);
+  }
```

Comment:
```suggestion
  :host([active]) {
    background-color: var(--mdc-navitem-active-background-color);
  }
  
  :host([disabled]) {
    color: var(--mdc-navitem-disabled-color);
    background-color: var(--mdc-navitem-disabled-background-color);
  }
```

If you interchange the order, you don't need the extra `:host([active][disabled])` to be applied